core/lib/data/network/download/download_client.dart

```diff
@@ -28,14 +35,72 @@ class DownloadClient {
     return (responseBody as ResponseBody);
   }
 
-  Future<String?> downloadImageAsBase64(String url) async {
+  Future<String?> downloadImageAsBase64(
```

Comment:
IMO, you should define a flag to config compress or not. WDYT?

Reply:
That is not necessary. We already have a check size flag.

Reply:
oke